### ISSUE TITLE
Release Request

### ISSUE THREAD

**Issue by cleithner-comcast** [modularml/stack-pr#43]
I tried to figure out if I could submit this in Discussions since it's not an issue, but didn't see a button. I was just wondering if there was a release planned anytime soon. I've been hand applying changes to pipx installations but there have been a lot of neat fixes since last release that would be nice to distribute.

**Comment by ZolotukhinM** [comment]
Yes, I'm planning to push a new release some time this week, sorry for the delay! :)

**Comment by ZolotukhinM** [comment]
Version 0.1.2 has been released, please try it out and let us know if you hit any issues!

**Comment by ZolotukhinM** [comment]
There was an extra 0.1.3 release today to fix a bug that snuck in right before the 0.1.2 - please update to that if you have installed 0.1.2.
